### ISSUE TITLE
ModuleNotFoundError: No module named 'wakatime'

### ISSUE THREAD

**Issue by ilyar** [wakatime/gedit-wakatime#5]
*env*
```
pip --version
pip 19.2.1 from /usr/local/lib/python3.6/dist-packages/pip (python 3.6)
python --version
Python 3.6.8
```

*install.py*
```
host:~/download/gedit-wakatime-1.0.1
@user $ sudo ./install.py 
Traceback (most recent call last):
  File "./install.py", line 32, in check_wakatime
    import wakatime
ModuleNotFoundError: No module named 'wakatime'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./install.py", line 56, in <module>
    install()
  File "./install.py", line 45, in install
    check_wakatime()
  File "./install.py", line 37, in check_wakatime
    pip.main(['install', 'wakatime'])
AttributeError: module 'pip' has no attribute 'main'
```

**Comment by ilyar** [comment]
*solution*

```
$ pip install wakatime && ./install.py
Wakatime plugin installation successful.
Now activate it in gedit Edit > Preferences > Plugins.
```

**Comment by alanhamlett** [comment]
Fixed with fe0b7e9fb1464af7aa25c20381d438c7e15e9184.
